sample-metadata-response.json

```diff
@@ -0,0 +1,4 @@
+{
+  "token": "cea5c8d7-fe73-4fbb-897c-7e22e3adc3e2",
```

Comment:
The idea here is that a token will be generated and given back immediately to the requestor. When the callback URL is hit with this JSON, the requestor can match the status with the token they are given.